vue/components/forms/personalization-form/reference.vue

```diff
@@ -140,13 +140,7 @@ export const Reference = {
              * The height (in pixels) of the image that is going to be used to
              * display the personalization initials.
              */
-            imageHeight: 300,
-
-            /**
-             * The radius of the border for the image that is going to be used in
-             * the display of the initials.
-             */
-            imageBorderRadius: "50%"

```

Comment:
Is anyone using this prop?

Reply:
this $data besides having a default "50%" was previously being set in `mounted` here https://github.com/ripe-tech/ripe-commons-pluginus/blob/master/vue/components/forms/personalization-form/reference.vue#L278
but now that logic is in the new computed

Reply:
I understand that, my question was if there's a `v-bind:image-border-radius` somewhere

Reply:
> I understand that, my question was if there's a `v-bind:image-border-radius` somewhere

it's only used to pass this `imageBorderRadius` to `<initials-image>` https://github.com/search?q=org%3Aripe-tech+v-bind+image-border-radius&type=code